docs/src/getting_started.md

```diff
@@ -11,7 +11,18 @@ using Pkg
 Pkg.add(ClimaLand)
 ```
 
-## Running a simple soil model
+## Steps to run a model
+
+To run a model, you will always need to do these steps:
+- Import dependencies. You need some libraries to run the model.
+- Chose Float32 or Float64.
+- Import parameters. You need some constants and parameters values to run the model.
```

Comment:
we can note that parameters are specified via toml files, see page X for details